_data/research.yaml

```diff
@@ -19,7 +19,7 @@
 - title: Ultra-coarse-grained simulations of protein self-assembly
   subtitle: 
   group: featured
-  image: images/software.png
+  image: images/research-ultra-cg.png
   link: 
   description: 
```

Comment:
Add a one line description of your research. You can see examples on me and Mayukh's pull request

Reply:
Also, don't end this sentence with a punctuation mark to be consistent with Mayukh and me.